tests/platform_tests/test_environment.py

```diff
@@ -418,10 +418,10 @@ def test_yaml(self):
 
         # get raw environment after
         # ruamel parser only used in py2.6+
-        if sys.version_info < (2,6):
-            env_file = os.path.join(self.project_config, "env", "test_post_update_old_parser.yml")
+        if sys.version_info < (2, 6):
+            env_file = os.path.join(self.project_config, "env", "post_update", "test_post_update_old_parser.yml")
```

Comment:
These tests load tess.yml and insert an empty engine in them and without  a location, which is an error for a real file. Moving these files out of the root folder solves the issue.